signer/local/local_test.go

```diff
@@ -489,7 +489,12 @@ func TestOverrideSubject(t *testing.T) {
 
 	s := newCustomSigner(t, testECDSACaFile, testECDSACaKeyFile)
 
-	certPEM, err := s.Sign(signer.SignRequest{Hostname: "localhost", Request: string(csrPEM), Subject: req})
+	certPEM, err := s.Sign(signer.SignRequest{
+		Hosts:   []string{"127.0.0.1", "localhost"},
+		Request: string(csrPEM),
+		Subject: req,
+	})
+
```

Comment:
Consider add a tests block below to verify the hosts are present in the right place.